readme.txt

```diff
@@ -33,13 +33,13 @@ In the plugin settings you can edit the title of the plugin, and enable or disab
 
 == Screenshots ==
 
-1. This is the meta box right to the post editor.
-1. This is the posts page.
-1. These are the widget options.
-1. This is the control panel in frontend for logged in users.
-1. This is the widget in frontend for not logged in users.
-1. This is the statistics page which displays the top 25 posts.
-1. This is the settings page.
+1. This is the meta box right to the post editor. Post Worktime Logger will add a new interactive box to your edit screen, allowing you to keep track of how much time you spend on your alterations. You may stop the current counting whenever you need to do something else, or reset it altogether, for example, if you want to restart your work from scratch.
+2. This is the posts page. The plugin changes your post list, granting you an additional column that shows the time spent on any particular post.
```

Comment:
...Well, I've used a [validator](https://wordpress.org/plugins/about/validator/) that says otherwise, but I will do these changes anyway 😉